src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
     public class ConsoleLogger : BaseLogger, IAnalysisLogger
```

Comment:
>ConsoleLogger [](start = 17, length = 13)

Make sure that only error *notifications* are not suppressed when --quiet is passed. *error* results are suppressed on the console when the flag is passed. related let's call this argument --quiet-console